app/views/development_metrics/merge_time/_project.erb

```diff
@@ -0,0 +1,5 @@
+<%= render partial: 'shared/line-chart',
```

Comment:
Should we rename this partial to something more generic? It is used for all types of metrics, not just `project` metrics right?

Reply:
I'm not sure what you mean

Reply:
Sorry, I realize now that it was misleading 😄 

I meant the name of the partial you are editing (`merge_time/_project.rb`). Should it still have that name, given that you are using it for department metrics?